zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1869,6 +1869,78 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /resources/{resourceKey}:
+    get:
+      tags:
+        - Resource
+      summary: Get resource
+      description: |
+        Returns a deployed resource.
+        :::note
+        Currently, this endpoint only supports RPA resources.
+        :::
```

Comment:
👍 It took me some time to verify whether these note admonitions are correctly rendered by our docs, because they aren't used in the current published docs, but they work!

🔧 I think we can increase the visibility to [INFO or WARNING](https://docusaurus.io/docs/markdown-features/admonitions).

Reply:
Actually, they are already used in this `rest-api.yaml` file for a couple of endpoints that are marked as `alpha` 🙂 (e.g. https://docs.camunda.io/docs/8.7/apis-tools/camunda-api-rest/specifications/query-decision-definitions-alpha/).

I've increased the visibility to `INFO` (I wouldn't say that it's worth a `WARNING`).